src/app/templates/app/submit_new_license.html

```diff
@@ -151,7 +135,7 @@
 
```

Comment:
Check License Diff is commented out - why? (did this ever work, I can't remember?)

Reply:
Yes, this was disabled in April this year following this issue - https://github.com/spdx/spdx-online-tools/issues/355 

We can take a call to re-enable this on this page after this gets in https://github.com/spdx/spdx-online-tools/pull/392 which improves matching